src/app/session/session-state.service.ts

```diff
@@ -27,16 +10,23 @@ export class SessionState {
   session = new BehaviorSubject<Session>(new Session());
 
   get budgetItems(): Observable<BudgetItem[]> {
-    const sessionObservable = this.session.asObservable();
+    const session$ = this.session.asObservable();
```

Comment:
Why did we switch back to session$? I thought we were using sessionObservable, since that is what it is.

Reply:
It's common nomenclature in rxJS (the observable/behaviourSubject/pipe/subscribe library) to use $ to mean "Observable". I originally wasn't a big fan, but a good example of why it's a nice shorthand is in add-item.page.ts, where we want the observable (categories$) and a local copy of the most recent value from the observable (categories). The $ means you don't have to name "categories" "_categories", or "categories$" "categoriesObservable". It's more concise. If you prefer the longer version I'm cool with that too. The reason I also changed it here, when we don't have a local copy of session is just for consistency

Reply:
Ok sure thing. If thats the nomenclature, im fine with it, was just wondering.